net-mail/vpopmail/vpopmail-5.4.33-r5.ebuild

```diff
@@ -0,0 +1,247 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils fixheadtails qmail
```

Comment:
Please forgive me if I'm missing something but are you using eutils anywhere?

Reply:
probably a leftover from previous versions